apps/test/netsim/NetSimLogEntry.js

```diff
@@ -112,11 +114,16 @@ describe("NetSimLogEntry", function () {
   it ("can extract packet data based on standard format", function () {
     var logEntry = new NetSimLogEntry(null, {
       packet: '000100100011010001010110'
```

Comment:
Went with
```
packet: '0001 0010 0011 0100 01010110'
```
since in this context the binary gets "minified" before it's operated on.